framework/test/unit/specs/modules/keys/keys_module.spec.ts

```diff
@@ -11,3 +11,523 @@
  *
  * Removal or modification of this copyright notice is prohibited.
  */
+import { LiskValidationError } from '@liskhq/lisk-validator';
+import { codec } from '@liskhq/lisk-codec';
+import { getRandomBytes, hash } from '@liskhq/lisk-cryptography';
+import { Account, GenesisBlock, Transaction, transactionSchema } from '@liskhq/lisk-chain';
+import { objects as ObjectUtils } from '@liskhq/lisk-utils';
+import { KeysModule } from '../../../../../src/modules/keys/keys_module';
+import { createFakeDefaultAccount, StateStoreMock } from '../../../../utils/node';
+import * as fixtures from './fixtures.json';
+import { GenesisConfig } from '../../../../../src';
+import { genesisBlock as createGenesisBlock } from '../../../../fixtures/blocks';
+import { AccountKeyAsset } from '../../../../../src/modules/keys/types';
+
+// eslint-disable-next-line import/order
+import cloneDeep = require('lodash.clonedeep');
```

Comment:
cloneDeep should be replaced by cloneDeep from `lisk-utils`

Reply:
Done